README.md

```diff
@@ -29,214 +40,54 @@ This Connector implements the [Data Connector Spec](https://github.com/hasura/nd
+### Step 2: Configure the connector
 
-# run the code generation (using CLI flags)
-docker run --rm -v ./:/etc/connector/ ghcr.io/hasura/ndc-open-api-lambda:v0.1.1 update --open-api ${url to open API document}
+Once you have an initialized supergraph and subgraph, run the initialization command in interactive mode while
+providing a name for the connector in the prompt:
 
-# with baseUrl (using env vars)
-docker run --rm -v ./:/etc/connector/ -e NDC_OAS_DOCUMENT_URI=${url to open API document} -e NDC_OAS_BASE_URL=http://demoapi.com/ ghcr.io/hasura/ndc-open-api-lambda:v0.1.1 update
-
-# with baseUrl (using CLI flags)
-docker run --rm -v ./:/etc/connector/ ghcr.io/hasura/ndc-open-api-lambda:v0.1.1 update --open-api ${url to open API document} --base-url http://demoapi.com/
+```bash
+ddn connector init <connector-name> -i
 ```
 
-## Build and Run
-
-You can use the Open API Connector via Docker or via the bundled CLI.
-
-### Using Docker
+#### Step 2.1: Choose `hasura/openapi` from the list
 
-Clone the repository and run the following commands to build the Docker image that can then be used for code generation.
-
-```
-cd ndc-open-api-lambda
+#### Step 2.2: Choose a port for the connector
 
-# build the docker image with the name `ndc-oas-lambda` and tag `latest`
-docker build -t ndc-oas-lambda:latest .
+The CLI will ask for a specific port to run the connector on. Choose a port that is not already in use or use the
+default suggested port.
 
-# get command documentation/help
-docker run --rm ndc-oas-lambda:latest update -h
+#### Step 2.3: Provide the env vars required for the connector
 
-# when running the code generation, the container will expect either of the following two things
-# 1. The OpenAPI document is mounted at `/etc/connector/` as `swagger.json`. OR
-# 2. The link to the OpenAPI document is provided via the env var `NDC_OAS_DOCUMENT_URI`
-# Please note that mounting a directory at /etc/connector/ is required in both cases, since that is where the code is generated
+Environment variables that can be used to configure the connector are listed under [Supported Environment Variables](./docs/documentation.md#supported-environment-variables)
 
-# **** START: Examples ****
-# run the code generation
-docker run --rm -v ./:/etc/connector/ -e NDC_OAS_DOCUMENT_URI=${url to open API document} ndc-oas-lambda:latest update
+## Step 3: Introspect the connector
 
-# with baseUrl
-docker run --rm -v ./:/etc/connector/ -e NDC_OAS_DOCUMENT_URI=${url to open API document} -e NDC_OAS_BASE_URL=http://demoapi.com/ ndc-oas-lambda:latest update
-
-# **** END: Examples ****
-
-# start the NodeJS Lambda Connector
-docker run --rm -p 8080:8080 -v ./:/etc/connector ndc-oas-lambda:latest
+```bash
+ddn connector introspect <connector-name>
 ```
 
-NOTE: You can also pass CLI flags with values to the Docker Container instead of environment variables
-
-### Using the CLI
+This will generate the required HML and TS files
```

Comment:
```suggestion
This will generate the required metadata and TypeScript files.
```

Since we haven't introduced the concept of HML yet, let's just use a more generic / accessible term.